engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -335,11 +335,11 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
   }
 
   private def validateNamespaceWithImports(w: WorkflowSourceFilesWithDependenciesZip): ErrorOr[WdlNamespaceWithWorkflow] = {
-    def getMetadatae(importsDir: Path, prefix: String = ""): List[(String, Path)] = {
-      importsDir.children.toList flatMap {
+    def getMetadatae(importsDir: Path, prefix: String = ""): Seq[(String, Path)] = {
```

Comment:
Please don't change return types from concrete to generic. If anything, go the other direction